lib/manageiq/appliance_console/database_replication_primary.rb

```diff
@@ -10,7 +10,7 @@ def initialize
       self.database_name     = "vmdb_production"
       self.database_user     = "root"
       self.database_password = nil
-      self.primary_host      = LinuxAdmin::NetworkInterface.new(NETWORK_INTERFACE).address
+      self.primary_host      = network_interfaces.first&.address
```

Comment:
NOTE this will only work reliably with a single non-loopback interface, but this is just the default presented to the user and they can enter any address that they want plus our appliances only ship by default with a single NIC.

In the future I think a `#default?` attribute or something like that which checks which dev is used for the default route would let us pick the interface most likely to be the "right one" :tm:

Reply:
I think that's fine.  There was no guarantee before that eth0 was the default route.